torchrl/objectives/a2c.py

```diff
@@ -383,7 +388,11 @@ def get_entropy_bonus(self, dist: d.Distribution) -> torch.Tensor:
             entropy = dist.entropy()
         except NotImplementedError:
             x = dist.rsample((self.samples_mc_entropy,))
-            entropy = -dist.log_prob(x).mean(0)
+            if isinstance(dist, CompositeDistribution):
+                log_prob = dist.log_prob(x).get(self.tensor_keys.sample_log_prob)
+            else:
+                log_prob = dist.log_prob(x)
+            entropy = -log_prob.mean(0)
```

Comment:
what if the dist is wrapped (eg via distributions.TransformedDistribution or smth like that)?
Shouldn't we check if `log_prob` is a tensordict instead? That would be more generic